app/models/protocol.rb

```diff
@@ -32,8 +33,19 @@ def max_streak
     rewards.last
   end
 
+  def stop_measurement
+    measurements.find(&:stop_measurement?)
+  end
+
   private
 
+  def at_most_one_stop_measurement
+    stop_measurements = measurements.select(&:stop_measurement?)
+    return if stop_measurements.nil? || stop_measurements.length <= 1
```

Comment:
return if `stop_measurements.blank?`

Want daar is het voor

Reply:
`<= 1`, dat is niet voldoende met `.blank?` of bedoel je de nil? ik heb van de nil een .blank gemaakt